elasticsearch/examples/kubesphere/README.md

```diff
@@ -137,17 +70,17 @@ uninstall:
 
 ```
 
-#### install
+#### Install
 
 ```
 make install [-e NAMESPACE=xxx]
 ```
 
-#### patch
+#### Patch
 
-Skip this step if status of the pods is healthy
+Skip this step if all the states of the pods are normal.
```

Comment:
Skip this step if all the states of the pods are normal.
=>
Skip this step if the states of all the pods are normal.